tests/Test.ADAL.NET.Unit/NonInteractiveTests.cs

```diff
@@ -127,10 +127,60 @@ public void Initialize()
         }
 
         [TestMethod]
+        [Description("Cloud Audience Urn Test")]
+        public async Task CloudAudienceUrnTest()
```

Comment:
Changes: "Cloud Audience Urn Test" to test the wsTrustRequest passing it a generic cloud audience urn (ex. "urn:federation:Blackforest"). "Cloud Audience Urn Null Test" to test wsTrustRequest passing it a null value for the cloud audience. 
Added a "Ws-Trust Request Test" to test WsTrustRequest, passing it a test constant value ("urn:federation:MicrosoftOnline"). Also, a "Ws-Trust Request Generic Cloud Urn Test, to test WsTrustRequest, passing it a test constant value ("urn:federation:Blackforest").